data/json/mapgen/cs_private_park.json

```diff
@@ -5,95 +5,76 @@
     "om_terrain": [ "cs_private_park" ],
     "weight": 1000,
     "object": {
-      "fill_ter": "t_floor",
+      "predecessor_mapgen": "forest_thick",
       "rows": [
-        "..xxxxxxxxxxxxxxxxxxxx.,",
-        ".WWWWWWWWW-XX-WWWWWWWWW.",
-        ",W......T.PxxP.s,p.p..W,",
-        ",W.f.p.f..oxxo..f..T..W.",
-        ".W.Y..p...oxxo.A.~dpA.W.",
-        ".Wo..T...T.xx.d.p.f.,.W.",
-        ",W..f.YfA.dxxxxxxxxxT.W,",
-        ",W~.,.dwwwAxxxxxxxxxY.W.",
-        ".WP..TwwwwwwwwwA.pxxY,W.",
-        ",W.f.AwwwwwwwwwwwAxxp.W,",
-        ".W.d.xxxxxxxxxxxxxxx..W.",
-        ".W~.Txxxxxxxxxxxxxxx.TW.",
-        ".W.,.xxwwwwwwwwwwfY.p,W.",
-        ",W.f.xxAwwwwwwwwA.Y.d.W,",
-        ".W.Y.xx,.wwwwwwwf.Tf,pW.",
-        ".W.T.xx.Y.T.wwT.f..A.pW.",
-        ".W..fxx...Y..p...~.T,.W,",
-        ",Wdddxxd~o~....P.fY...W.",
-        ".WzzzDDzzzo.T..f...,ooW.",
-        ".WazS   Lz~.Y.,pY.d.T.W.",
-        ",W C   21G,,.f...T.,..W,",
-        ".WtzBE  1z@@T.P...Y,.YW.",
-        ",WWWWWWWWWWWWWWWWW-X-WW.",
-        "..,,..,.....,......,..,."
+        "...........ss...........",
+        ".#########-XX-#########.",
+        ".#        f__f        #.",
+        ".#        f__f        #.",
+        ".#        f__fffffff  #.",
+        ".#        f________ff #.",
+        ".#        ff________f #.",
+        ".#       WWfffffff__f #.",
+        ".#      WWWWWWW  f__f #.",
+        ".#  fffWwwwwwwWWWf__f #.",
+        ".# ff__==========___f #.",
+        ".# f___==========__ff #.",
+        ".# f__fWwwwwwwwwWfff  #.",
+        ".# f__fWWwwwwwwWW     #.",
+        ".# f__f WWwwwwwW      #.",
+        ".# f__f  WWwwWWW      #.",
+        ".# f__f   WWWW        #.",
+        ".# f__f               #.",
+        ".#||DD|G||            #.",
+        ".#a|,,S,L|            #.",
+        ".#,C,,,,1G            #.",
+        ".#t|BE,21|            #.",
+        ".#################-X-##.",
+        "........................"
       ],
+      "flags": [ "ERASE_ALL_BEFORE_PLACING_TERRAIN" ],
+      "palettes": [ "park_nature_palette" ],
       "terrain": {
-        ",": "t_dirt",
-        "-": "t_chainfence_h",
-        ".": "t_grass",
-        " ": "t_floor",
-        "x": "t_sidewalk",
-        "w": "t_water_dp",
-        "z": "t_wall_wood",
-        "A": "t_tree_willow",
+        "-": "t_chainfence",
+        ".": "t_region_groundcover_urban",
+        "f": "t_region_groundcover_urban",
+        ",": "t_floor",
+        "1": "t_floor",
+        "2": "t_floor",
+        "a": "t_floor",
+        "t": "t_floor",
+        "B": "t_floor",
+        "E": "t_floor",
+        "L": "t_floor",
+        "S": "t_floor",
+        "s": "t_sidewalk",
+        "|": "t_wall_wood",
         "D": "t_door_locked_peep",
         "C": "t_door_c",
         "G": "t_curtains",
-        "P": "t_tree_cherry",
```

Comment:
Honestly it look like choice of trees might have been intentional here. A private park harboring fancy trees (which may include fruit trees) makes sense but I have no idea how you could specify that without hardcoding locations.

Reply:
![privatepark](https://user-images.githubusercontent.com/44979050/158534408-96c01b99-9f0d-48dd-a2e8-6b471ad065da.png)
Re-added willow and cherry trees.